libs/ui-watt/src/lib/foundations/color/_colors.import.scss

```diff
@@ -0,0 +1,89 @@
+@use 'sass:map';
```

Comment:
Question: Do you think renaming this file to `_colors.util.scss` or `_colors.api.scss` will make it more clear as to what it does? Or maybe split it into two files - one with API functions and one with color maps.

Reply:
I've taken the naming from angular material, the file does only contain the color map now